refinery/ui/source/js/data-set-import/partials/metadata-table.html

```diff
@@ -31,14 +31,14 @@
             Can have additional columns used to characterize data
           </li>
           <li>
-            Can have the columns ordered arbitrary
+            Can have the columns ordered arbitrarily
```

Comment:
"Can have the columns ordered arbitrarily" is clearer, so would leave the change as is.